### DISCUSSION TITLE
GitHub::API::ValidationFailedError

### DISCUSSION THREAD

**Discussion by jhon9392** [Homebrew/discussions#4732] [Tap maintenance and Homebrew development]
### Output of `brew config`

```shell
brew config                                                                        ──(Mon,Aug21)─┘
HOMEBREW_VERSION: 4.1.5
ORIGIN: https://github.com/Homebrew/brew
HEAD: b8b46501e4caa1cad75b30627c123127ab09a6a7
Last commit: 7 days ago
Core tap JSON: 21 Aug 12:35 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 10
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 14.0.3 build 1403
Git: 2.41.0 => /opt/homebrew/bin/git
Curl: 8.1.2 => /usr/bin/curl
macOS: 13.5-arm64
CLT: 14.3.1.0.1.1683849156
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
brew doctor                                                                        ──(Mon,Aug21)─┘
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Homebrew/homebrew-cask was not tapped properly! Run:
  rm -rf "/opt/homebrew/Library/Taps/homebrew/homebrew-cask"
  brew tap homebrew/cask

Warning: Some installed kegs have no formulae!
This means they were either deleted or installed manually.
You should find replacements for the following formulae:
  tools

Warning: Some installed formulae are deprecated or disabled.
You should find replacements for the following formulae:
  jsonschema
  postgresql@10

Warning: You have unlinked kegs in your Cellar.
Leaving kegs unlinked can lead to build-trouble and cause formulae that depend on
those kegs to fail to run properly once built. Run `brew link` on these:
  python-certifi
  tools
  pycparser
  cffi
  pyyaml
  python-cryptography

Warning: You have an unnecessary local Cask tap.
This can cause problems installing up-to-date casks.
Please remove it by running:
  brew untap homebrew/cask

Warning: Some taps are not on the default git origin branch and may not receive
updates. If this is a surprise to you, check out the default branch with:
  git -C $(brew --repo homebrew/cask) checkout
```

### Description of issue

i am trying to install custom Formula but there is an error while installing i see below error
/opt/homebrew/Cellar/kairos-awssamlapi-base/0.12.1_2/libexec/bin/pip
install
-r
/private/tmp/kairos-awssamlapi-base-20230821-28271-24vi78/requirements.txt

If reporting this issue please do so to (not Homebrew/brew or Homebrew/homebrew-core):
  kairos/tools

/opt/homebrew/Library/Homebrew/utils/github/api.rb:337:in `raise_error': Validation Failed: [{"message"=>"The listed users and repositories cannot be searched either because the resources do not exist or you do not have permission to view them.", "resource"=>"Search", "field"=>"q", "code"=>"invalid"}] (GitHub::API::ValidationFailedError)
        from /opt/homebrew/Library/Homebrew/utils/github/api.rb:263:in `open_rest'
        from /opt/homebrew/Library/Homebrew/utils/github.rb:178:in `search'
        from /opt/homebrew/Library/Homebrew/utils/github.rb:182:in `search_results_items'
        from /opt/homebrew/Library/Homebrew/utils/github.rb:36:in `search_issues'
        from /opt/homebrew/Library/Homebrew/utils/github.rb:62:in `issues_for_formula'
        from /opt/homebrew/Library/Homebrew/exceptions.rb:502:in `fetch_issues'
        from /opt/homebrew/Library/Homebrew/exceptions.rb:497:in `issues'
        from /opt/homebrew/Library/Homebrew/exceptions.rb:553:in `dump'
        from /opt/homebrew/Library/Homebrew/brew.rb:151:in `rescue in <main>'
        from /opt/homebrew/Library/Homebrew/brew.rb:139:in `<main>'
/opt/homebrew/Library/Homebrew/formula.rb:2741:in `block in system': Failed executing: /opt/homebrew/Cellar/kairos-awssamlapi-base/0.12.1_2/libexec/bin/pip install -r /private/tmp/kairos-awssamlapi-base-20230821-28271-24vi78/requirements.txt (BuildError)
        from /opt/homebrew/Library/Homebrew/formula.rb:2677:in `open'
        from /opt/homebrew/Library/Homebrew/formula.rb:2677:in `system'
        from /opt/homebrew/Library/Taps/kairos/homebrew-tools/Formula/kairos-awssamlapi-base.rb:37:in `install'
        from /opt/homebrew/Library/Homebrew/build.rb:177:in `block (3 levels) in install'
        from /opt/homebrew/Library/Homebrew/extend/kernel.rb:499:in `with_env'
        from /opt/homebrew/Library/Homebrew/build.rb:139:in `block (2 levels) in install'
        from /opt/homebrew/Library/Homebrew/formula.rb:1420:in `block in brew'
        from /opt/homebrew/Library/Homebrew/formula.rb:2908:in `block (2 levels) in stage'
        from /opt/homebrew/Library/Homebrew/extend/kernel.rb:499:in `with_env'
        from /opt/homebrew/Library/Homebrew/formula.rb:2907:in `block in stage'
        from /opt/homebrew/Library/Homebrew/resource.rb:128:in `block (2 levels) in unpack'
        from /opt/homebrew/Library/Homebrew/download_strategy.rb:113:in `chdir'
        from /opt/homebrew/Library/Homebrew/download_strategy.rb:105:in `stage'
        from /opt/homebrew/Library/Homebrew/resource.rb:124:in `block in unpack'
        from /opt/homebrew/Library/Homebrew/mktemp.rb:75:in `block in run'
        from /opt/homebrew/Library/Homebrew/mktemp.rb:75:in `chdir'
        from /opt/homebrew/Library/Homebrew/mktemp.rb:75:in `run'
        from /opt/homebrew/Library/Homebrew/resource.rb:228:in `stage_resource'
        from /opt/homebrew/Library/Homebrew/resource.rb:123:in `unpack'
        from /opt/homebrew/Library/Homebrew/resource.rb:97:in `stage'
        from /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/forwardable.rb:230:in `stage'
        from /opt/homebrew/Library/Homebrew/formula.rb:2887:in `stage'
        from /opt/homebrew/Library/Homebrew/formula.rb:1413:in `brew'
        from /opt/homebrew/Library/Homebrew/build.rb:133:in `block in install'
        from /opt/homebrew/Library/Homebrew/extend/kernel.rb:499:in `with_env'
        from /opt/homebrew/Library/Homebrew/build.rb:128:in `install'
        from /opt/homebrew/Library/Homebrew/build.rb:229:in `<main>'

**Comment by p-linnane** [comment]
From the error message you posted:
>If reporting this issue please do so to (not Homebrew/brew or Homebrew/homebrew-core):
kairos/tools
